lib/elasticity/multi_search.rb

```diff
@@ -1,5 +1,7 @@
 module Elasticity
   class MultiSearch
+    include Enumerable
```

Comment:
this can be removed.  I was changing it into a collection object but with the redesign it no longer is.  The each method was only being used to make the enumerable include work.